src/os/win32/elevate.cpp

```diff
@@ -16,34 +16,36 @@
 
 #include "os/elevate.h"
 
-static std::string JoinArguments(std::vector<std::string> arguments) {
-  std::ostringstream result;
+static std::wstring JoinArguments(std::vector<std::wstring> arguments) {
+  std::wstring result = L"";
 
-  std::copy(arguments.begin(), arguments.end(),
-            std::ostream_iterator<std::string>(result, " "));
+  for (const std::wstring argument : arguments) {
```

Comment:
I resorted to this loop given making `stringstream` work with `wstring` was not very straightforward. I'll keep trying to crack that nut in the mean time.